### DISCUSSION TITLE
ideas

### DISCUSSION THREAD

**Discussion by sda17** [1Remote/1Remote#295] [Ideas]
Already congratulations for the work done.
It would be interesting to have:
- a keyboard shortcut to navigate between open tabs
- the possibility to split the window, horizontally and vertically
- to be able to import *.rdp to facilitate the configuration

have a nice day

**Comment by VShawn** [comment]
Hi Thx for your ideas

> a keyboard shortcut to navigate between open tabs

Actually I have make effect on this but failed, shortcut was redirect to remote desk.

We can not capture the combine key on local machine when you are in a RDP or SSH session. 

If you have a VNC session, you will found you can navigate between open tabs by CTRL + 1\2\3

It seems the only way to get the combine key is to implement a global keyboard hooks in Windows. This behave like virus  I don't want to do this.

see also: https://github.com/1Remote/PRemoteM/issues/40


> the possibility to split the window, horizontally and vertically

I do not intend to support this feature at this time since Windows 10 offers a number of ways to split your screen into multiple window:

https://user-images.githubusercontent.com/10143738/192969531-9e09bab2-6568-4906-ba7b-274e44298179.png

> to be able to import *.rdp to facilitate the configuration

This feature can be implemented. The only trouble is that you can't read the password from the *.rdp file.

**Reply by sda17** [reply]
Thanks for the clarification and the quick feedback.

For the import, I have indeed found a way to import the entire content of the rdp file in the additional parameters that the application offers. This allowed me to run the chrome application remotely. I couldn't do it in RemoteApp mode.
Thanks again

**Comment by majkinetor** [comment]
> It seems the only way to get the combine key is to implement a global keyboard hooks in Windows. This behave like virus I don't want to do this.

I don't think this will work. I tried with Autohotkey and hotkey gets ignored. There might be a way to do this as far as I remember, someobdy mention it, but can't remember now where.... this feature would be great for other reasons too, you sometimes need to active host command while on remote...

> since Windows 10 offers a number of ways to split your screen into multiple window:

This is not the same. For example tmux and Conemu support this. Good thing is that you could save layout and recall it later the way you want. However, this is not trivial to do and it can be left for future, especially since it can be scripted or used with more advanced window manager...

> to be able to import *.rdp to facilitate the configuration

While we are at this, we should also have _export_. We have it now but its not very userfriendly (opens it in notepad and then you can save it).

**Comment by haowani** [comment]
Add scp protocol  ideas

**Comment by majkinetor** [comment]
I added shortcut to ticket #405. Importing RDP is possible. Splitting window is cool option, but not trivial and requires careful UI redesign, so its out of scope for this time.
